subsys/net/lib/aws_fota/include/aws_fota_json.h

```diff
@@ -31,34 +31,43 @@ extern "C" {
  * @brief Parse a given AWS IoT DescribeJobExecution response JSON object.
  *	  More information on this object can be found at https://docs.aws.amazon.com/iot/latest/developerguide/jobs-api.html#mqtt-describejobexecution
  *
- * @param[in] job_document JSON formated string of an AWS Job execution
+ * @param[in] job_document JSON formated string of an AWS IoT
```

Comment:
Many changes in this commit overwrite changes from the previous commit. See if some of them can be put into that commit instead. Or maybe they should be one commit.

Reply:
I could just squash them.